base_product_mass_addition/models/product_product.py

```diff
@@ -0,0 +1,46 @@
+# © 2014 Today Akretion
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# @author Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# @author Pierrick Brun <pierrick.brun@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    qty_to_process = fields.Float(
+        compute="_compute_process_qty",
+        inverse="_inverse_set_process_qty",
+        help="Set this quantity to create a new line "
+        "for this product or update the existing one.",
+    )
+
+    def _inverse_set_process_qty(self):
+        parent_model = self.env.context.get("parent_model")
+        parent_id = self.env.context.get("parent_id")
+        if parent_model:
+            parent = self.env[parent_model].browse(parent_id)
+            for product in self:
+                quick_line = parent._get_quick_line(product)
+                if quick_line:
+                    parent._update_quick_line(product, quick_line)
+                else:
+                    parent._add_quick_line(product, quick_line._name)
+
+    def _compute_process_qty(self):
+        if not self.env.context.get("parent_id"):
```

Comment:
Isn't it possible to get this with a parameter instead of context ?

Reply:
It is called from `compute=` on a field, I don't think this would be possible (or I missed something ?)

Reply:
@phuctranerp I was trying to migrate `stock_picking_quick` using this branch for the needed dependency in order to test it, and I've found that this method causes an error. `stock_picking_quick` already implements its own `_compute_process_qty` method, and even if inside it `qty_to_process` is always filled, an error is fired. Only if `_compute_process_qty` code in `base_product_mass_additions` is changed in order to write an initial value in `qty_to_process` for every record the error is not fired.

Since the computed field is not stored, every implementation of computed method should assign a value for every record (v13 requirement), even for the `super()` method.

Reply:
You can test it using https://github.com/solvosci/stock-logistics-workflow/tree/13.0-mig-stock_picking_quick

Reply:
Hello @dalonsod
Thanks for your feedbacks, I think the implementation of this method is originally not a "computation" method, so the author didnt implement the computation method. This allows to trigger the _inverse_set_process_qty method only. The migration is v12 as-is. Do you have any recommendation ?

Reply:
In case of `stock_picking_quick` `_compute_process_qty` is finally used. Making this change in `base_product_mass_addition` method worked for me (making a trivial assignment):

```suggestion
        for product in self:
                product.qty_to_process = 0
```
I don't really know if this change could fire an unexpected behavior